spring/boot-autoconfigure/src/test/resources/config/application-sslTest.yml

```diff
@@ -0,0 +1,8 @@
+armeria:
+  ports:
+    - port: 0
+      protocol: HTTP
+    - port: 0
+      protocol: HTTPS
+  ssl:
+    use-self-signed-certification: true
```

Comment:
I can't find this property defined anywhere. Could you let me know where it is?

Reply:
Maybe I editted copied version of `Ssl` and the code is already gone during build time.